scripts/qa_crowbarsetup.sh

```diff
@@ -3016,8 +3016,8 @@ function deploy_single_proposal
             fi
             ;;
         murano)
-            if ! iscloudver 7plus; then
-                echo "Murano is SOC 7+ only. Skipping"
+            if ! ( iscloudver 7 || iscloudver 8 ) ; then
```

Comment:
reversed logic ?!

Reply:
Please see https://github.com/SUSE-Cloud/automation/pull/2795 where this got requested.